Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -381,6 +381,7 @@ + (NSString *)pushTokenKey
 + (void)start
 {
     [self startWithUserId:nil userAttributes:nil responseHandler:nil];
+    [LPFileManager clearCacheIfNeeded];
```

Comment:
since start is just calling `startwith...` using default parameters, we should put this method at the beginning of the method that does all the work